infrastructure/src/main/java/org/corfudb/infrastructure/management/FullyConnectedClusterRecommendationEngine.java

```diff
@@ -176,6 +184,9 @@ public ClusterRecommendationStrategy getClusterRecommendationStrategy() {
                                     .add(currentNode));
         }
 
+        log.debug("Converted view of the cluster to a symmetric view of connectivity failures: {}",
```

Comment:
another way would be to log.info whenever there is any output. On that trigger you can log both input and output. Hope is that would be seldom enough that you can have it as info.

Reply:
@medhavidhawan If we just log when healed/failed nodes are being proposed, it can be INFO. However my thought was to provide more logs at DEBUG level even the alg doesn't output any healed or failed. Do you think it's ok to keep the DEBUG logs and add two INFO level logs for when outputs are not empty?

Reply:
As we discussed offline, added the info level logs for when here are non empty proposed healed or failed nodes.